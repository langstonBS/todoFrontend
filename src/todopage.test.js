import React from 'react';
import { shallow } from 'enzyme';
import ToDoPage from './todopage';

test('renders correctly', () => {
  const wrapper = shallow(<ToDoPage />);
  expect(wrapper).toMatchSnapshot();
});