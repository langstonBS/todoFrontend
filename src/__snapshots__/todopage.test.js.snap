// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <form
    onSubmit={[Function]}
  >
    <input
      onChange={[Function]}
      value=""
    />
    <button>
      click
    </button>
  </form>
  <ul>
    <li
      key="{\\"id\\":1,\\"todo\\":\\"wash cow\\",\\"compleet\\":false}"
      onClick={[Function]}
    >
      wash cow
    </li>
    <li
      key="{\\"id\\":2,\\"todo\\":\\"wash bird\\",\\"compleet\\":false}"
      onClick={[Function]}
    >
      wash bird
    </li>
    <li
      key="{\\"id\\":3,\\"todo\\":\\"wash plane\\",\\"compleet\\":false}"
      onClick={[Function]}
    >
      wash plane
    </li>
    <li
      key="{\\"id\\":4,\\"todo\\":\\"wash fatcst\\",\\"compleet\\":false}"
      onClick={[Function]}
    >
      wash fatcst
    </li>
    <li
      key="{\\"id\\":5,\\"todo\\":\\"wash idk\\",\\"compleet\\":true}"
      onClick={[Function]}
    >
      wash idk
    </li>
    <li
      key="{\\"id\\":6,\\"todo\\":\\"just fucking do somthing\\",\\"compleet\\":false}"
      onClick={[Function]}
    >
      just fucking do somthing
    </li>
  </ul>
</div>
`;
